src/pipx/main.py

```diff
@@ -543,6 +543,7 @@ def setup(args):
     else:
         logging.basicConfig(level=logging.WARNING, format="%(message)s")
 
+    logging.info(f"Pipx version is {__version__}")
```

Comment:
this is a total nit, but could `pipx` be changed to lowercase?

Reply:
Done!  
I thought I'd copied the casing from somewhere else, but apparently I did not.  :)